Changed build target to .NET Standard 2.0 🚀

sven-n opened the pull request:
Currently, this project targets the .NET Framework. However, we want to migrate to .NET 5 and beyond.

I analyzed the code and found no issues when it targets .NET Standard 2.0, which is compatible to both .NET Framework and newer .NET versions.

If you have access to the published nuget package (#10, @bhattvishal ?), it would be great to get an updated package as well.

Thank you 😀

sven-n committed:
Update .gitignore

Prevent _ReSharper.Caches to be checked in

sven-n committed:
Changed build target to .net standard 2.0

This improves compatibility. We want to be compatible with newer .net (core) versions.

kallemooo commented:
The automatic build system did not like the commit.
Can you please check the error?

For the nuget package it is an unofficial upload, that I did not do. I will try again to get the rights to replace that version,

sven-n committed:
fixed build

The name of the build target of the pre-build event has to be "PreBuild" instead of "BeforeBuild"

sven-n commented:
The target name of the pre-build had to be changed. It should work now.

sven-n commented:
Hi, any news? You can re-run the build check, btw ;-)

kallemooo commented on Asap2/Asap2.csproj:
```diff
@@ -64,70 +35,33 @@
+  </PropertyGroup>
+
   <ItemGroup>
-    <Compile Include="Asap2Tree\Asap2Base.cs" />
```
Why are all project files not included anymore?

sven-n replied:
I had to change the project file format to the new SDK format. With this format, all files (including subfolders) are included by convention by default.
More info here: https://learn.microsoft.com/en-us/dotnet/core/project-sdk/overview

kallemooo commented:
> Hi, any news? You can re-run the build check, btw ;-)

The change causes the build to fail.
Please check this.

Did you build using the command line tools on your computer as the CI build does?

I cannot check this as I do not have windows installed any longer.

sven-n commented:
Oh, didn't expect it still fails :-( I built it on my machine using vs 2019. I'll check it with the command line when I'm back at the office.
